conference.js

```diff
@@ -71,7 +72,7 @@ const eventEmitter = new EventEmitter();
 let room;
 let connection;
 let localAudio, localVideo;
-let initialAudioMutedState = false, initialVideoMutedState = false;
+let initialAudioMutedState = false;
```

Comment:
What's driving me crazy about that field is yet another source of truth for video muted. We have muted in base/media, videoMuted field in conference and then the video track can have it's own muted status. On the other hand we'll have startWithVideoMuted option and we could simply say you can't toggle until the conference starts to make our life easier.